From c0374887c9805fedd15cc2a24b0b3601e0e1202a Mon Sep 17 00:00:00 2001
From: Mike Christie <michaelc@cs.wisc.edu>
Date: Wed, 29 Jul 2015 04:23:55 -0500
Subject: [PATCH] target: add lio rbd to makefile/Kconfig
References: fate#318836
Patch-mainline: Not yet, SES2 clustered LIO/RBD

Add lio rbd backend module to target Makefile and Kconfig.

Signed-off-by: Mike Christie <michaelc@cs.wisc.edu>
Acked-by: David Disseldorp <ddiss@suse.de>

---
 drivers/target/Kconfig  |    6 ++++++
 drivers/target/Makefile |    1 +
 2 files changed, 7 insertions(+)

--- a/drivers/target/Kconfig
+++ b/drivers/target/Kconfig
@@ -39,6 +39,12 @@ config TCM_USER2
 	process to handle requests. This is version 2 of the ABI; version 1
 	is obsolete.
 
+config TCM_RBD
+	tristate "TCM/RBD Subsystem Plugin for Linux/RBD"
+	help
+	Say Y here to enable the TCM/RBD subsystem plugin for Ceph RBD
+	access.
+
 source "drivers/target/loopback/Kconfig"
 source "drivers/target/tcm_fc/Kconfig"
 source "drivers/target/iscsi/Kconfig"
--- a/drivers/target/Makefile
+++ b/drivers/target/Makefile
@@ -23,6 +23,7 @@ obj-$(CONFIG_TCM_IBLOCK)	+= target_core_
 obj-$(CONFIG_TCM_FILEIO)	+= target_core_file.o
 obj-$(CONFIG_TCM_PSCSI)		+= target_core_pscsi.o
 obj-$(CONFIG_TCM_USER2)		+= target_core_user.o
+obj-$(CONFIG_TCM_RBD)		+= target_core_rbd.o
 
 # Fabric modules
 obj-$(CONFIG_LOOPBACK_TARGET)	+= loopback/
